### ISSUE TITLE
MS-SSIM Usage

### ISSUE THREAD

**Issue by Auth0rM0rgan** [jorge-pessoa/pytorch-msssim#5]
Hi @jorge-pessoa,

Thanks a lot for sharing the code. I have some doubts about the usage of your implementation.

This is the way I am using the msssim in my code as a loss function.
```
loss_SSIM_A = pytorch_msssim.msssim(G_BA(real_A), real_A, normalize=True)
loss_SSIM_B = pytorch_msssim.msssim(G_AB(real_B), real_B, normalize=True)
loss_SSIM = (loss_SSIM_A + loss_SSIM_B) / 2
```
The result value until epoch 20 is betwwen 0.85 - 0.97 and then decrease to 0.65 until 0.81 , Is it fine or I need to define the threshold to maximize the value of MSSSIM as you did in max-ssim.py ?

1. Am I using the msssim correctly? because in the max-ssim.py you add the minus sign; should I do the same or not and what is the reason of using the minus sign?

   `msssim_out = -loss_func(img1, img2)`

Thanks in advance!

**Comment by jorge-pessoa** [comment]
1. You need to use a minus sign as PyTorch optimizers will optimize the network in the direction of minimizing the objective function, (in this case, MS-SSIM). As you usually want to reach an output that maximizes MS-SSIM to a value as close of 1 as possible, you need to optimize for the symmetric of MS-SSIM in order to transform it into a minimization problem.
